acceptance/bundle/debug/script

```diff
@@ -1,4 +1,4 @@
 $CLI bundle validate --debug 2> full.stderr.txt
 grep -vw parallel full.stderr.txt > out.stderr.txt
-grep -w parallel full.stderr.txt | sort_lines.py > out.stderr.parallel.txt
+grep -w parallel full.stderr.txt | sed 's/[0-9]/0/g' | sort_lines.py > out.stderr.parallel.txt
```

Comment:
If this should apply to the time only, then the pattern should be more narrow, and not include `/g`.

Reply:
Narrow patterns indeed should be used for global replacements.

However, since this is scoped to one test (even more so - one file within one test) and we capture the whole output I think it's okay to go with simpler regex, even if ends up capturing more. This is especially true for regexes that output the same amount of characters that were consumed  - if there were issues, we'll see them.

I'm actually starting to prefer simpler patterns in test-scoped configs. Another example: https://github.com/databricks/cli/pull/2282/files#diff-572952aa7a8b85bafdba5774fdf61454a3bb9fbb4f9a6aeaa09d936e34687c1b